modin/core/storage_formats/pandas/utils.py

```diff
@@ -18,73 +18,36 @@
+def compute_chunksize(index_len, num_splits, default_block_size=None):
     """
-    Get the most equal chunksize possible based on length and number of splits.
-
-    Parameters
-    ----------
-    length : int
-        The length to split (number of rows/columns).
-    num_splits : int
-        The number of splits.
-
-    Returns
-    -------
-    int
-        Computed chunksize.
-    """
-    return (
-        length // num_splits if length % num_splits == 0 else length // num_splits + 1
-    )
+    Compute the number of elemenents (rows/columns) to include in each partition.
 
-
-def compute_chunksize(
-    df,
-    num_splits,
-    default_block_size=None,
-    axis=None,
-):
-    """
-    Compute the number of rows and/or columns to include in each partition.
+    Chunksize is defined the same for the axes.
 
     Parameters
     ----------
-    df : pandas.DataFrame
-        DataFrame to split.
+    index_len : int
+        Element count in a index.
     num_splits : int
-        Number of splits to separate the DataFrame into.
+        The number of splits.
     default_block_size : int, optional
         Minimum number of rows/columns in a single split.
         If not specified, the value is assumed equal to ``MinPartitionSize``.
-    axis : int, optional
-        Axis to split across. If not specified - split accros both axes.
 
     Returns
     -------
-    int if axis was specified, tuple of ints otherwise,
-        If axis is 1 or 0, returns an integer number of rows/columns to split the
-        DataFrame. If axis is None, returns a tuple containing both.
+    int
+        Integer number of rows/columns to split the DataFrame will be returned.
     """
-    default_block_size = (
-        MinPartitionSize.get() if default_block_size is None else default_block_size
-    )
-    if axis == 0 or axis is None:
-        row_chunksize = get_default_chunksize(len(df.index), num_splits)
-        # Take the min of the default and the memory-usage chunksize first to avoid a
-        # large amount of small partitions.
-        row_chunksize = max(1, row_chunksize, default_block_size)
-        if axis == 0:
-            return row_chunksize
-    # We always execute this because we can only get here if axis is 1 or None.
-    col_chunksize = get_default_chunksize(len(df.columns), num_splits)
-    # Take the min of the default and the memory-usage chunksize first to avoid a
-    # large amount of small partitions.
-    col_chunksize = max(1, col_chunksize, default_block_size)
-    if axis == 1:
-        return col_chunksize
+    if default_block_size is None:
+        default_block_size = MinPartitionSize.get()
 
-    return row_chunksize, col_chunksize
+    chunksize = index_len // num_splits
+    if index_len % num_splits:
+        chunksize += 1
+    # chunksize shouldn't be less that `default_block_size` to avoid a
+    # large amount of small partitions.
+    return max(chunksize, default_block_size)
```

Comment:
This can be 0 if user sets something silly for `MinPartitionSize` and `chunksize` is less than `num_splits`. That is why we had a 1 value here before, so that the chunksize is always 1 or greater. It needs a comment, I agree, but I think we should keep the 1 here.

```suggestion
    return max(1, chunksize, default_block_size)
```

Reply:
Maybe we should add a check for assigned values for `MinPartitionSize`? For example, check if `value > 0`.

Reply:
I think we should do both. It doesn't hurt to always ensure positive number here in case other checks go wrong.

Reply:
ok, I see. @vnlitvinov what do you think?

Reply:
dunno, I would rather set an assert early on checking if variables have sensible values, rather than on anything weird default to some edge case

Reply:
I added the check in `MinPartitionSize.put`. 

Although we have the ability to pass the erroneous value through `default_block_size` argument, it is not used or tested anywhere. Let's remove it. @vnlitvinov @devin-petersohn thoughts?

Reply:
I think `default_block_size` is not a good name, but it is something we can plan to use in the future with better partitioning during the query planning phase. 

I see two options: (Option 1) is an `assert` here, (Option 2) is `max(1, ...)`. I don't care either way, but I don't want an unchecked and untested value here.

Reply:
`assert` is added.